.ddev/commands/web/wp-reset

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+show-help() {
+	echo -e "\nReset the WordPress database and reinstall WordPress"
+	echo -e "\tExample: ddev wp-reset"
+	echo -e "\nRemove all plugins --clean-plugins"
+	echo -e "\tExample: ddev wp-reset --clean-plugins\n"
+}
+
+remove-plugins() {
+	echo "Deactivating all plugins..."
+	wp plugin deactivate --all --path=.ddev/wordpress
+	
+	echo "Deleting all plugins..."
+	wp plugin delete --all --path=.ddev/wordpress
+}
+
+reset-database() {
+	echo "Resetting database..."
+	wp db reset --yes --path=.ddev/wordpress
+
+	if [ "$1" = "clean-plugins" ]; then
+		remove-plugins
+	fi
+
+	echo "Installing WordPress..."
+	wp core install \
+		--url=wp-test-site.ddev.site \
+		--title="WooCommerce PayPal Payments" \
+		--admin_user=admin \
```

Comment:
maybe should the same config as in https://github.com/woocommerce/woocommerce-paypal-payments/blob/11f33651b48e538c9ee21ae95b22e460a54bd076/.ddev/commands/web/orchestrate.d/10_wp_install.sh#L27